docs/troubleshooting.md

```diff
@@ -77,7 +77,7 @@ kubectl exec -it promtail-bth9q -- /bin/ash
 
 Once connected, verify the config in `/etc/promtail/promtail.yml` is what you expected
 
-Also check `/var/log/positions.yaml` and make sure promtail is tailing the logs you would expect
+Also check `/var/run/promtail/positions.yaml` and make sure promtail is tailing the logs you would expect
```

Comment:
```suggestion
Also check `/var/log/positions.yaml` (default to `/var/run/promtail/positions.yaml` if you're using helm) and make sure promtail is tailing the logs you would expect
```

Reply:
Again, the helm chart has it at `/run/promtail/positions.yaml`. But other than that sounds agreeable.